sample-iwa/src/manifest.webmanifest

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Simple IWA", // Displayed name instead of isolated-app://gibberish URL
+  "id": "/",            // Must be "/"
+  "scope": "/",         // Must be "/"
+  "start_url": "/",     // Must be non-empty and refer to a resource in the bundle
+  "version": "0.1.0",   // Compulsory to be non-null.
+  "display": "standalone", // Should be one of the windowed ones
```

Comment:
> Should be one of the windowed ones

I don't get this comment.

Reply:
Probably talking about how the app will be open, so as part of a browser window or as a standalone app with it's own window.

I would maybe write "The app should open as a standalone window"